utils/installer/lv-config.example.lua

```diff
@@ -66,6 +66,24 @@ O.lang.php.diagnostics.signs = true
 O.lang.php.diagnostics.underline = true
 O.lang.php.filetypes = {"php", "phtml"}
 
+--LaTeX
```

Comment:
Could you add this line below the comment
-- Reference: https://github.com/latex-lsp/texlab/blob/master/docs/options.md

Reply:
Done